src/xrEngine/Device_Initialize.cpp

```diff
@@ -50,8 +50,17 @@ void CRenderDevice::Initialize()
         const Uint32 flags = SDL_WINDOW_BORDERLESS | SDL_WINDOW_HIDDEN |
             SDL_WINDOW_RESIZABLE | SDL_WINDOW_OPENGL;
 
-        m_sdlWnd = SDL_CreateWindow("S.T.A.L.K.E.R.: Call of Pripyat", 0, 0, 640, 480, flags);
+        if (psDeviceFlags.test(rsRGL))
```

Comment:
Seems like at this point `psDeviceFlags` hasn't yet been initialized, so this test will fail always.

Reply:
Oh.. True.. I'll think on it..

Reply:
We can try to move initialization stages in `xrEngine/main.cpp`